libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp

```diff
@@ -389,37 +389,25 @@ void AP_Volz_Protocol::update()
                 continue;
             }
 
-            // @LoggerMessage: VOLZ
-            // @Description: Volz servo data
-            // @Field: TimeUS: Time since system startup
-            // @Field: I: Instance
-            // @Field: Dang: desired angle
-            // @Field: ang: reported angle
-            // @Field: pc: primary supply current
-            // @Field: sc: secondary supply current
-            // @Field: pv: primary supply voltage
-            // @Field: sv: secondary supply voltage
-            // @Field: mt: motor temperature
-            // @Field: pt: pcb temperature
-            AP::logger().WriteStreaming("VOLZ",
-                "TimeUS,I,Dang,ang,pc,sc,pv,sv,mt,pt",
-                "s#ddAAvvOO",
-                "F000000000",
-                "QBffffffhh",
-                AP_HAL::micros64(),
-                i + 1, // convert to 1 indexed to match actuator IDs and SERVOx numbering
-                telem.data[i].desired_angle,
-                telem.data[i].angle,
-                telem.data[i].primary_current,
-                telem.data[i].secondary_current,
-                telem.data[i].primary_voltage,
-                telem.data[i].secondary_voltage,
```

Comment:
We do loose logging of the secondary current and voltage, they could be added as extra fields in the future.